tests/e2e/cucumber.conf.js

```diff
@@ -0,0 +1,19 @@
+const config = {
+  // environment
+  baseUrlOcis: process.env.BASE_URL_OCIS ?? 'https://host.docker.internal:9200',
```

Comment:
Can it be renamed to ...
```suggestion
  baseUrlOcis: process.env.OCIS_URL ?? 'https://host.docker.internal:9200',
```